uaclient/api/tests/test_api_u_pro_security_status_reboot_v1.py

```diff
@@ -0,0 +1,25 @@
+import mock
+import pytest
+
+from uaclient.api.u.pro.security.status.reboot.v1 import reboot_status
```

Comment:
I think generally the python function name should match the api endpoint/module name. How about `u.pro.security.status.reboot_required.v1` and `reboot_required`?